portalnet/src/events.rs

```diff
@@ -59,6 +63,18 @@ impl PortalnetEvents {
                         None => warn!("History event handler not initialized!"),
                     };
                 }
+                ProtocolId::Beacon => {
+                    match &self.beacon_overlay_sender {
+                        Some(tx) => {
+                            if let Err(err) = tx.send(request) {
+                                error!(
+                                    "Error sending discv5 talk request to beacon network: {err}"
+                                );
+                            }
+                        }
+                        None => warn!("Beacon event handler not initialized!"),
```

Comment:
Seems like this case might be worthy of an `error!`? eg a mistake has been made somewhere in the user's config / our architecture if a user tries to send a beacon msg and the subnetwork is not available